packages/@okta/vuepress-site/docs/reference/api/rulesets/index.md

```diff
@@ -0,0 +1,979 @@
+}' "https://${yourOktaDomain}/api/v1/policies"
+```
+
+##### Response Types
+
+HTTP 204:
+[RuleSet object](#rule-set-object)
+
+### Activate a Rule Set
+
+<ApiOperation method="post" url="/api/v1/policies/${rulesetId}/lifecycle/activate" />
+
+##### Request Parameters
+
+The rule set id described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/lifecycle/activate"
+```
+
+##### Response Types
+
+HTTP 204:
+*No Content is returned when the activation is successful.*
+
+### Deactivate a Rule Set
+
+<ApiOperation method="post" url="/api/v1/policies/${rulesetId}/lifecycle/deactivate" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/lifecycle/deactivate"
+```
+
+##### Response Types
+
+HTTP 200:
+*No Content is returned when the deactivation is successful.*
+
+## Rule Operations
+
+### Get Rules in a Rule Set
+
+<ApiOperation method="get" url="/api/v1/policies/${rulesetId}/rules" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules"
+```
+
+##### Response Types
+
+HTTP 200:
+[Rule object](#rule-object)
+
+### Create a rule
+
+<ApiOperation method="post" url="/api/v1/policies/${rulesetId}/rules" />
+
+##### Request Parameters
+
+The policy ID described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "type": "Okta:SignOn",
+  "name": "Example Rule",
+  "status": "ACTIVE",
+  "default": false,
+  "priority": 0,
+  "conditions": [
+     {
+        "key": "Okta:User",
+        "op": "IN_LIST",
+        "value": [
+           "00u1pdsiceJZLRJMSEUA"
+        ]
+     }
+  ],
+  "action": "ALLOW",
+  "requirement: [
+     "verificationMethod": {
+         "type": "ASSURANCE",
+         "factorMode": "1FA",
+         "constraints": [
+            {
+              "knowledge": {
+                 "types": [
+                   "password"
+                 ]
+              }
+            }
+         ],
+         "reauthenticateIn": "PT1M"
+     }
+  ]
+}' "https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules"
+```
+
+##### Response Types
+
+HTTP 200:
+[Rule object](#rule-object)
```

Comment:
Does this return only the Rule you created, or all the Rules in that particular Rule Set?

Reply:
It will return the rule you created